### ISSUE TITLE
mouseup event outside the dialog closes the dialog

### ISSUE THREAD

**Issue by anthonytordillos** [PolymerElements/paper-dialog#11]
Could we instead listen for clicks? The current behavior is problematic if you try to select text within the dialog and accidentally release the click outside the dialog.

**Comment by Mygod** [comment]
+1

**Comment by tjsavage** [comment]
@notwaldorf would you mind taking a look at this one?

**Comment by anthonytordillos** [comment]
Any update here?

**Comment by samccone** [comment]
So I did some digging around, and we are not actually listening for mouseup, rather we are bound to the full `click` event.

![screen shot 2015-08-21 at 9 42 50 am](https://cloud.githubusercontent.com/assets/883126/9409646/fb8222e2-47e9-11e5-9532-7bd6f76efeee.png)

The :bug: bug in this case (if we want to support this feature) is in `iron-overlay-behavior`

![screen shot 2015-08-21 at 9 44 44 am](https://cloud.githubusercontent.com/assets/883126/9409666/1689e610-47ea-11e5-84e9-aba33057d23f.png)

To fix this, we might want to listen for the mousedown and mouseup event and then check to see if the e.target on mousedown was in the dialog initially, if so abort the cancel operation.

---

TLDR; this is a "bug" / feature in iron-overlay-behavior. We should be able to close this issue and open one there.

thoughts @notwaldorf ?

**Comment by notwaldorf** [comment]
@cdata is the `IronOverlayBehavior` expert here so I'm going to pass the hot potato his way.

**Comment by anthonytordillos** [comment]
Ping?

**Comment by valdrinkoshi** [comment]
Tested on https://elements.polymer-project.org/elements/paper-dialog?view=demo:demo/index.html&active=paper-dialog
Could not reproduce, so closing this

**Comment by Ewanuk** [comment]
I'm seeing this myself, is there no solution or workaround? Pretty frustrating behaviour.
